indico/modules/rb_new/__init__.py

```diff
@@ -40,6 +43,27 @@ def _sidemenu_items(sender, event, **kwargs):
                            icon='location')
 
 
+@signals.users.merged.connect
+def _merge_users(target, source, **kwargs):
+    BlockingPrincipal.merge_users(target, source, 'blocking')
+    Blocking.find(created_by_id=source.id).update({Blocking.created_by_id: target.id})
```

Comment:
Let's use `XXX.query.filter_by(...)` instead of `XXX.find()` - less stuff to update when we finally get rid of it

Reply:
This was just copy-paste from the other file.